### PR TITLE
[SW-1595] Document How to Generate Prediction Contributions from an Existing MOJO

### PR DESCRIPTION
**Pull request by jakubhava**

### CODE REVIEW

**Comment by mn-mikke** [comment]
@jakubhava WDYT about adding a note about  what versions of MOJO files contain information about contributions in case users wanted to use older MOJOs?

**Comment by jakubhava** [comment]
Good catch @mn-mikke , added

File: doc/src/site/sphinx/tutorials/shap_values.rst [hunk 1]
```diff
@@ -108,3 +114,91 @@ To get SHAP values(=contributions) from H2OXGBoost model, please do:
...
+
+            val path = '/path/to/mojo.zip'
+            settings = H2OMOJOSettings(withDetailedPredictionCol=True)
+            model = H2OMOJOModel.createFromMojo(path, settings)
+
+        Run Predictions
+
+        .. code:: python
+
+            model.transform(testingDF).show(truncate = False)
```
Comment (mn-mikke) [inline doc/src/site/sphinx/tutorials/shap_values.rst]: `predictions = model.transform(testingDF)`

File: doc/src/site/sphinx/tutorials/shap_values.rst [hunk 2]
```diff
@@ -108,3 +114,91 @@ To get SHAP values(=contributions) from H2OXGBoost model, please do:
...
+
+            val path = '/path/to/mojo.zip'
+            val settings = H2OMOJOSettings(withDetailedPredictionCol=true)
+            val model = H2OMOJOModel.createFromMojo(path, settings)
+
+        Run Predictions
+
+        .. code:: scala
+
+            val predictions = model.transform(testingDF).show(false)
```
Comment (mn-mikke) [inline doc/src/site/sphinx/tutorials/shap_values.rst]: remove `.show(false)`

File: doc/src/site/sphinx/tutorials/shap_values.rst [hunk 3]
```diff
@@ -108,3 +114,91 @@ To get SHAP values(=contributions) from H2OXGBoost model, please do:
...
+
+        Show contributions
+
+        .. code:: scala
+
+            predictions.select("detailed_prediction.contribution").show()
```
Comment (mn-mikke) [inline doc/src/site/sphinx/tutorials/shap_values.rst]: `"detailed_prediction.contribution"` => `"detailed_prediction.contributions"`?

File: doc/src/site/sphinx/tutorials/shap_values.rst (hunk 1 above)
Comment (jakubhava) [reply doc/src/site/sphinx/tutorials/shap_values.rst]: 👍  thanks!

File: doc/src/site/sphinx/tutorials/shap_values.rst (hunk 2 above)
Comment (jakubhava) [reply doc/src/site/sphinx/tutorials/shap_values.rst]: 👍

File: doc/src/site/sphinx/tutorials/shap_values.rst (hunk 3 above)
Comment (jakubhava) [reply doc/src/site/sphinx/tutorials/shap_values.rst]: 👍

**Comment by jakubhava** [comment]
Thanks for the review!

**Pull request merged**